lod/variable-display.md

```diff
@@ -0,0 +1,174 @@
+
+### IDLx Textbox
+
+Show lab instructions based on text a user enters into a textbox. The user will enter text (variable value) into the text box that corresponds with a variable name/variable value that was set using one of the methods above and any lab instructions or instruction elements that use that variable name will be shown.
+
+>[!knowledge] It is recommended to provide the variable values in the lab instructions, so that lab users can enter a variable value that corresponds to a variable name. If a user enters text that does not correspond with a variable name, no lab instructions will be shown based on the user's input. 
+
+1. From the lab profile, click **Edit Instructions**. 
+1. Click the @lab menu and choose `@lab.TextBox(yourVariableNameHere)`.
+1. Insert this into the section of the lab instructions where you want the drop-down menu to appear. 
+1. When a user enters text (variableValue) into the text box, any lab instructions or instruction elements that use that variable name will be shown.
+
+### IDLx Drop-down Menu
+
+Display a drop-down menu in lab instructions, where users can select an option. The options shown in the drop-down menu are variable values that were configured when the drop-down was created in the previous section. 
+
+1. From the lab profile, click **Edit Instructions**. 
+1. Click the @lab menu and choose token for the drop-down menu you created. If you created multiple drop-down menus, they will be listed here.
+1. Insert this into the section of the lab instructions where you want the drop-down menu to appear. 
+
+If you have not created a drop-down menu, please see the [IDLx Drop-down Menu section above](#idlx-drop-down-menu).
+
+
+### Dynamic Includes From an External Repository
+
+Lab instructions can be dynamically displayed based on the variable set in the lab and the variable value appended to the file name in the external repository. 
+
+1. From the lab profile, click **Edit Instructions**.
+1. If the lab variable is not already set in your lab, use [one of the methods above](#-setting-variables) to set a lab variable. 
+1. In the lab instructions, enter a Markdown include with a lab variable appended: `!instructions[](url-@lab.Variable(variableName).md)`
+1. Replace `url` with the url to the raw file in your external repository.
+1. Replace `variableName` with the name of the variable you wish to use.
+1. Choose an option:
+    - Insert a drop-down menu or textbox is the lab instructions that corresponds to the variable. These can be found in the @lab menu at the top of the window. 
+    - Assemble a Launch Command URL with the lab variable appended, to launch a lab with lab instructions displayed based on the variable value appended.
+
+>[!knowledge] Example:
+> 
+> **External repository file name**: 
+> 
+> `sample1-en.md`.
+>
+>**Include syntax with variable in file name**: 
+> 
+> `!instructions[(https://raw.githubusercontent.com/user1/sample1-@lab.Variable(language).md)`
```

Comment:
I believe the syntax should be !instructions[](raw.githubusercontent.com/.../...)